src/main/java/io/antmedia/AppSettings.java

```diff
@@ -1333,6 +1335,23 @@
 	@Value( "${" + SETTINGS_HLS_ENCRYPTION_KEY_INFO_FILE +":#{null}}")
 	private String hlsEncryptionKeyInfoFile;
 	
+	/*
+	 * JWT Control Type
+	 * It can be default or jwks
+	 * It's "default" by default
```

Comment:
What's the default? I know what it is but please make it easy to understand for everyone.

Reply:
I don't think we need to have additional `jwtControlType`. If `jwksURL` is not null, use JWKS. If it's null and JWT is enabled, use the JWT